scripts/emulator-testing/emulators/emulator.ts

```diff
@@ -123,18 +123,42 @@ export abstract class Emulator {
     });
   }
 
+  findDataConnectConfigDir() {
+    let path = './';
+    const files = fs.readdirSync(path);
+    console.log(files);
```

Comment:
nit: can `console.log()` be removed? it looks like it may have been added during development for debugging purposes.

Reply:
There are a few other occurrences of `console.log()` in this file to consider as well.

Reply:
Deleted most of the ones I added, but the rest should be helpful for test failure debugging in the future.